### ISSUE TITLE
New pipe suggestion: Wrap

### ISSUE THREAD

**Issue by maxime1992** [danrevah/ngx-pipes#112]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ x ] feature request
```

**Current behavior**
I find myself doing that kind of things sometimes:

`<div [style.background-image]="'url(' + myBgUrl + ')'">`

**Expected behavior**

It's not optimized for change detection and doing that instead would be great:

`<div [style.background-image]="'myBgUrl | wrap:'url(':')'">`

If this is likely going to be accepted, I can probably raise a PR.

**Comment by danrevah** [comment]
I had some exams lately so I wasn't available then sorry for the late response.

This seems to me like a good idea, would you like to make a PR?

**Comment by danrevah** [comment]
Good job. released to version `2.3.0`.
